### ISSUE TITLE
Healthcheck issue for nemollm-inference-microservice

### ISSUE THREAD

**Issue by vbelouso** [NVIDIA-NIM-Agent-Blueprints/digital-human#7]
Hi
I attempted to run `nemollm-inference-microservice` via `Docker Compose`
The container image is `nvcr.io/nim/meta/llama3-8b-instruct:1.0.3`
I noticed that the [healthcheck](https://github.com/NVIDIA-NIM-Agent-Blueprints/digital-human/blob/main/deploy/docker-compose-nim-ms.yaml#L31) was unsuccessful.

I think this is because `curl` is not installed in the container image.

```shell
docker run --entrypoint curl nvcr.io/nim/meta/llama3-8b-instruct:1.0.3
```
Output: 
```text
unable to start container process: exec: "curl": executable file not found in $PATH: unknown.
```

But the same command works for `nvcr.io/nim/nvidia/nv-embedqa-e5-v5:1.0.1`

```shell
docker run --entrypoint curl nvcr.io/nim/nvidia/nv-embedqa-e5-v5:1.0.1
```
Output: 
```text
curl: try 'curl --help' or 'curl --manual' for more information
```

To resolve this issue temporarily, I replaced the curl-based healthcheck with a Python-based check:
```shell
test: ["CMD-SHELL", "python -c \"import urllib.request; urllib.request.urlopen('http://localhost:8000/v1/health/ready')\""]
```
Upon inspecting the container image history, I found that curl is initially installed but later removed:

```shell
TARGETARCH=amd64 /bin/sh -c apt-get update && apt-get install -y --no-install-recommends     gnupg2 curl ca-certificates &&     curl -fsSLO https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/${NVARCH}/cuda-keyring_1.0-1_all.deb &&     dpkg -i cuda-keyring_1.0-1_all.deb &&     apt-get purge --autoremove -y curl     && rm -rf /var/lib/apt/lists/* # buildkit 
```

Could you please address this issue by either installing `curl` or updating the healthcheck to avoid relying on `curl`?

Thank you!

**Comment by nreamaroon** [comment]
Hey @vbelouso, thanks for catching this! 

We've resolved this issue in 6a485707f4ead0207753eadd7e450b9bef5f595c so the healthcheck shouldn't fail anymore 🙂

**Comment by vbelouso** [comment]
Thanks @nreamaroon 
Closed by #9
